### DISCUSSION TITLE
여기에 올리는게 아니면 죄송합니다. 채팅 읽어주는 봇관련 문의 드려요

### DISCUSSION THREAD

**Discussion by LongMay123** [Lastorder-DC/chatreader-kor#29] [문의] [answered]
트위치에서 소소하게 방송하는 사람입니다. 

이번에 채팅읽어주는 봇을 쓰고있는데 어느날부터 기능을 하지 않습니다. 

원래는 크롬을 사용했는데 여러 브라우저로 돌아가면서 해봐도 안됩니다. 

그래서 다른 시연 영상들과 비교해보니 그분들 영상에는 왼쪽 위에 초기화 중이라는 메세지가 아니라 초기화 라는 메시지로 짙은 붉은 색이 뜨는데 저는 기다려봐도 옅은 붉은 색으로 초기화 중 이라는 메시지만 나옵니다.버튼을 누를수도 없게 되어 있고요. 

개인적으로는 이것이 문제인것 같은데 어떻게 해결할수 있나요? 

다국적언어를 한번에 읽어주는 봇은 라스트오더님것 밖에 없어서 많이 의존하고 있었습니다. 바쁘시겠지만 시간을 조금 내셔서 조언해주시면 감사하겠습니다.

**Comment by LongMay123** [comment]
사용하는OS는 윈도우 10입니다.

**Comment by dmc31a42** [comment]
#27 문제 때문에 그렇습니다. 해결될 때까지 잠시 기다려보세요

**Reply by LongMay123** [reply]
빠른 답변 감사합니다.

**Comment by Lastorder-DC** [comment, marked as answer]
#27 에서 수정되었습니다.
p.s.) 여기에 올리시는게 맞습니다. 😄

**Reply by LongMay123** [reply]
빠른 답변 감사합니다.
